pkgs/development/ocaml-modules/ocaml-migrate-parsetree/1.8.x.nix

```diff
@@ -8,7 +8,7 @@ buildDunePackage rec {
    pname = "ocaml-migrate-parsetree";
    version = "1.8.0";
 
-   useDune2 = lib.versionAtLeast ocaml.version "4.08";
+   duneVersion = if lib.versionAtLeast ocaml.version "4.08" then "3" else "1";
```

Comment:
@vbgl this seems to break coq-nix-toolbox for Coq 8.11 and 8.12 (using OCaml 4.10.2): https://github.com/coq-community/coq-nix-toolbox/actions/runs/4752181770/jobs/8442234540?pr=143

Reply:
Well, actually this is probably not the source of the breakage (I didn't run a bisect), I'm just noting the package seems broken.